documents/minutes/2019-08-06 WCA Board Meeting.md

```diff
@@ -21,8 +21,8 @@ The Board agreed to schedule quarterly Board meetings. Attendance at such meetin
 
 As a result of negative feedback regarding the timing for applications with the current round of Continental Championships, in conjunction with feedback from Ethan Pride regarding the difficulty of planning the World Championship in less than one year, the Board agreed to open application periods earlier and extend their length to offer organizational teams more time to apply and give the successful applicants more time to prepare. The following timetable was agreed upon:
 
-| **Type** | **Year** | **Location** | **Applications open** | **Applications close** | **Board decision** |
-| :--------- | :--------- | :--------- | :--------- | :--------- | :--------- |
+| Type | Year | Location&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Applications open | Applications close | Board decision |
```

Comment:
Whoopsie? What is this?

Reply:
Yeah, this is the result of a very long and tedious struggle to get the proper sizing of all table columns here. I tried just messing around with the number of dashes for a while here, but it doesn't seem to work well, because the table is too wide or something. This hack ended up making the table the most compact out of everything I tried. If you can find a way to keep it as compact as this without adding a bunch of spaces, then please do, haha. I'm not sure how to size it properly tbh.

Reply:
How does the table look if you don't add all the non-breaking spaces?

Reply:
The column is narrower and there's a line (or two) where text wraps because of that, which makes the table take up more space, when it doesn't have to.

Reply:
I'd argue that a table not being perfectly aligned is less ugly than manually squeezing in non-breakable spaces in the source code.

If you really, _really_, *really* wanted. there is a thing called "CSS" to solve styling problems for you ;-) But this should not be your concern now, just leave it without the nbsp and if the Board feels they want prettier tables in their not-so-super-relevant meeting minutes they can approach WST and we will figure something out